recipe/meta.yaml

```
@@ -9,21 +9,19 @@ source:
   sha256: edf8dbd19704685f7400c6fc3fcb875ab858b1e55382c7ec933efddff28b2332
 
 build:
-  number: 0
+  number: 1
   run_exports:
     - {{ pin_subpackage("aws-c-io", max_pin="x.x.x") }}
-  missing_dso_whitelist:  # [s390x]
-    - '$RPATH/ld64.so.1'  # [s390x]
 
 requirements:
   build:
     - cmake !=3.19.0,!=3.19.1
     - {{ compiler('c') }}
     - ninja-base
   host:
-    - aws-c-common 0.11.1
-    - aws-c-cal 0.8.5
-    - s2n 1.5.14 # [linux]
+    - aws-c-common 0.12.3
+    - aws-c-cal 0.9.2
+    - s2n 1.5.14  # [linux]
```

https://github.com/AnacondaRecipes/aggregate/blob/master/conda_build_config.yaml#L686
```suggestion
    - s2n {{ s2n }}  # [linux]
```